src/StaticWebAssetsSdk/Targets/Microsoft.NET.Sdk.StaticWebAssets.ServiceWorkerAssetsManifest.targets

```diff
@@ -19,8 +19,10 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <PropertyGroup>
     <ResolveStaticWebAssetsInputsDependsOn>$(ResolveStaticWebAssetsInputsDependsOn);_AddServiceWorkerAssets</ResolveStaticWebAssetsInputsDependsOn>
-    <GenerateComputedPublishStaticWebAssetsDependsOn>$(GenerateComputedPublishStaticWebAssetsDependsOn);_WritePublishServiceWorkerAssetsManifest</GenerateComputedPublishStaticWebAssetsDependsOn>
-    <GenerateComputedBuildStaticWebAssetsDependsOn>$(GenerateComputedBuildStaticWebAssetsDependsOn);_WriteBuildServiceWorkerAssetsManifest</GenerateComputedBuildStaticWebAssetsDependsOn>
+    <GenerateComputedPublishStaticWebAssetsDependsOn>$(GenerateComputedPublishStaticWebAssetsDependsOn)</GenerateComputedPublishStaticWebAssetsDependsOn>
+    <GenerateComputedBuildStaticWebAssetsDependsOn>$(GenerateComputedBuildStaticWebAssetsDependsOn)</GenerateComputedBuildStaticWebAssetsDependsOn>
+    <CompressFilesForPublishDependsOn>$(CompressFilesForPublishDependsOn);_WritePublishServiceWorkerAssetsManifest</CompressFilesForPublishDependsOn>
+    <CompressFilesDependsOn>$(CompressFilesDependsOn);_WriteBuildServiceWorkerAssetsManifest;</CompressFilesDependsOn>
```

Comment:
This is for now. We probably want to have the service worker come at a later stage so that it can see all the compressed assets paths too.

We can always manually compress the service worker and the manifest as an extra step.